src/weekly/WeeklyExporter.js

```diff
@@ -91,6 +91,10 @@ export class WeeklyExporter {
   }
 
   processRow (row: { [string]: string }) {
+   // todo quick fix for now
+    if (row[SUBSCRIPTION_NAME] === 'Subscription.Name') {
+      return
+    }
```

Comment:
this is because when we concatenate the introductory periods and regular subs streams the header is included in the point where the second file starts. Maybe we can do something to skip the first line of the second file when concatenating